WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -2027,7 +2027,9 @@ private enum ActionSheetStrings {
                                                comment: "Button title View product in store in Edit Product More Options Action Sheet")
     static let share = NSLocalizedString("Share", comment: "Button title Share in Edit Product More Options Action Sheet")
     static let promoteWithBlaze = NSLocalizedString("Promote with Blaze", comment: "Button title Promote with Blaze in Edit Product More Options Action Sheet")
-    static let delete = NSLocalizedString("Delete", comment: "Button title Delete in Edit Product More Options Action Sheet")
+    static let trashProduct = NSLocalizedString("productForm.bottomSheet.trashAction",
+                                                value: "Trash product",
```

Comment:
From PR testing instructions,

> Verify that the button on iOS for putting in the trash a product now reads “Move to Trash”.

Is "Trash product" the expected title?

Reply:
Sorry for being unclear, and yes. 
It's "Trash product" in the sheet, and "Move to trash" in the alert as mentioned here https://github.com/woocommerce/woocommerce-ios/issues/12271